backend/src/types.ts

```diff
@@ -87,44 +87,59 @@ export type KubeFastifyInstance = FastifyInstance & {
  * Common types, should be kept up to date with frontend types
  */
 
-export type OdhApplication = {
-  metadata: {
-    name: string;
```

Comment:
I believe name is required and should remain in the definition.

Reply:
i've updated OdhApplication, OdhDocument using the types generated from kpt [typegen](https://pkg.go.dev/github.com/googlecontainertools/kpt-functions-sdk/go/cmd/typegen) which will generate k8 types for a given cluster including the CRDs. I can add typegen into the build process at a later point.